Views/PersonPart.liquid

```diff
@@ -0,0 +1,21 @@
+{% comment %}
+    You see this template rendered if you delete the PersonPart.cshtml or you create a template with the same content
+    through the admin UI at Design/Templates. The name should be the file name without the extension.
+
+    Without the options.MemberAccessStrategy.Register<PersonPartViewModel>() in the startup you couldn't use the values
+    inside the viewmodel here, they would be null.
+
+    Note that the short date formatting is not available by default in liquid, so we are using a custom short date
+    format. Liquid default date formatter uses http://strftime.net/ format.
+
+    You can write custom filters that e.g. would convert DateTime to short date format you can see that below.
```

Comment:
I would rather put the `"convert DateTime to short date format you can see that below"` part of comment right above:
```
<div>{{ "Birth Date short date: {0}" | t: shortDate }}</div>
```

So the user can't miss it.